### ISSUE TITLE
Timeout in `itest/lnd_channel_policy_test`

### ISSUE THREAD

**Issue by hieblmi** [lightningnetwork/lnd#6746]
### Background

Running the isolated test `make itest icase=update_channel` even after increasing the from 30s to 3min.

```
Running integration tests with btcd backend.
rm -rf lntest/itest/*.log lntest/itest/.logs-*; date
Sun Jul 17 08:50:28 EDT 2022
EXEC_SUFFIX= scripts/itest_part.sh 0 1  -test.run="TestLightningNetworkDaemon/tranche.*/.*-of-.*/.*/update_channel" -test.timeout=60m 
/xxx/xxx/projects/lnd/lntest/itest/itest.test -test.v -test.run=TestLightningNetworkDaemon/tranche.*/.*-of-.*/.*/update_channel -test.timeout=60m -logoutput -goroutinedump -logdir=.logs-tranche0 -lndexec=/xxx/xxx/projects/lnd/lntest/itest/lnd-itest -btcdexec=/xxx/xxx/projects/lnd/lntest/itest/btcd-itest -splittranches=1 -runtranche=0
=== RUN   TestLightningNetworkDaemon
=== RUN   TestLightningNetworkDaemon/tranche00/08-of-105/btcd/update_channel_policy
    assertions.go:1786: 
        	Error Trace:	assertions.go:1786
        	            				lnd_channel_policy_test.go:26
        	            				lnd_channel_policy_test.go:490
        	            				test_harness.go:114
        	            				lnd_test.go:245
        	Error:      	Received unexpected error:
        	            	channel:4836abf9a7ddb6322b4039fa3f9b0844e39498842c0542f9c1075c90ed40eb47:0 policy not updated before timeout: [0360dc21dba6a66b69b8ea12ba8d4e6cfb4d25495b782049fcedd95c9a6164f5e5:time_lock_delta:22  min_htlc:1000  fee_base_msat:1600  fee_rate_milli_msat:123000000  max_htlc_msat:1000000] 
        	            	node state: {
        	            		"NodeID": 2,
        	            		"Name": "Carol",
        	            		"PubKey": "027148cb6c9c3f4e8134ce8bea2d323da7205ef0f8372fb8c1875a8c15dfdea6aa",
        	            		"OpenChans": {
        	            			"4836abf9a7ddb6322b4039fa3f9b0844e39498842c0542f9c1075c90ed40eb47:0": 3,
        	            			"6ab75d8e657b6837d5b6977725dd0c9b63a7e91edaaddda646025f0b89049bc0:0": 2,
        	            			"bf81b6a94a3bc3dcce89a833c68f4b1327d4e75da0a22845758366dbe363297f:0": 5
        	            		},
        	            		"ClosedChans": {},
        	            		"NodeCfg": {
        	            			"LogFilenamePrefix": "update_channel_policy",
        	            			"ExtraArgs": [
        	            				"--gossip.max-channel-update-burst=2",
        	            				"--gossip.channel-update-interval=24h"
        	            			],
        	            			"HasSeed": false,
        	            			"P2PPort": 5569,
        	            			"RPCPort": 5570,
        	            			"RESTPort": 5571,
        	            			"ProfilePort": 0,
        	            			"AcceptKeySend": true,
        	            			"AcceptAMP": false,
        	            			"FeeURL": "http://localhost:5560/fee-estimates.json"
        	            		}
        	            	}
        	Test:       	TestLightningNetworkDaemon/tranche00/08-of-105/btcd/update_channel_policy
        	Messages:   	error while waiting for channel update


=== CONT  TestLightningNetworkDaemon
    lnd_test.go:253: Failure time: 2022-07-17 08:52:02.774
--- FAIL: TestLightningNetworkDaemon (94.06s)
    --- FAIL: TestLightningNetworkDaemon/tranche00/08-of-105/btcd/update_channel_policy (92.52s)
```

### Your environment

0.15.0-beta

**Comment by yyforyongyu** [comment]
This is a known issue and will be fixed once #5940 is ready.

**Comment by hieblmi** [comment]
This turns out to be a local issue. The test runs fine on master. Closing the issue.
